chapter1/chapter1.tex

```diff
@@ -155,9 +158,9 @@ \subsection{Sample spaces and events}
 
 In general, this leaves us with the pair $ (\Omega, \mathcal{P}(\Omega)) $. For outcomes in a sample space,
 let us stress again an important difference, namely that $ \omega \in \Omega $ but 
-$ \{\omega\} \in \mathcal{A} $.
+$ \{\omega\} \in \mathcal{P}(\Omega)$.
```

Comment:
I think we can leave it like this since by contvention \mathcal(A) = \mathcal(P)(\Omega). But \omega \in \mathcal(A) is more general in case you want to use some other event space.